src/prefect/server/models/block_documents.py

```diff
@@ -260,52 +260,68 @@ async def read_block_document_by_name(
 
 
 @inject_db
-async def read_block_documents(
-    session: AsyncSession,
+def _apply_block_document_filters(
+    query,
     db: PrefectDBInterface,
-    block_document_filter: Optional[schemas.filters.BlockDocumentFilter] = None,
-    block_type_filter: Optional[schemas.filters.BlockTypeFilter] = None,
-    block_schema_filter: Optional[schemas.filters.BlockSchemaFilter] = None,
-    include_secrets: bool = False,
-    sort: Optional[
-        schemas.sorting.BlockDocumentSort
-    ] = schemas.sorting.BlockDocumentSort.NAME_ASC,
-    offset: Optional[int] = None,
-    limit: Optional[int] = None,
+    block_document_filter: schemas.filters.BlockDocumentFilter | None,
+    block_schema_filter: schemas.filters.BlockSchemaFilter | None,
+    block_type_filter: schemas.filters.BlockTypeFilter | None,
```

Comment:
uriel and I were just talking yesterday about how we wish we could use the "pip" or syntax in OSS 🙂

Reply:
Just a few more deprecation cycles!